repos/system_upgrade/el7toel8/libraries/tests/test_isccfg.py

```diff
@@ -343,5 +345,22 @@ def test_walk():
     assert 'dnssec-validation' not in state
 
 
+def test_empty_config():
+    """ Test empty configuration """
+
+    callbacks = {
+        'options': cb_state,
+        'dnssec-lookaside': cb_state,
+        'dnssec-validation': cb_state,
+    }
+    state = {}
```

Comment:
Remove state usage, make it default to None. Unlike views test it is not needed here.

Reply:
We can probably remove the `state` variable since it can be omitted from the `walk` call